mosviz/viewers/mos_viewer.py

```diff
@@ -69,6 +69,8 @@ def __init__(self, session, parent=None):
         self.textChangedAt = None
         self.mask = None
         self.cutout_wcs = None
+        self.level2_data = None
+        self.spec2d_data = None
```

Comment:
Where do these attributes get used?

Reply:
I should mention that I see that they're set below, but they don't appear to be referenced anywhere else

Reply:
They are only used in that instance, they are the only way to keep tabs for now. I did this to enable the level2 code, perhaps we can find a better solution when we refactor this class.